test/example.js

```diff
@@ -162,6 +162,8 @@ const functionWithFunction = (fn, number) => fn(Math.random() * number);
 
 const arrayExportExample = (num1, num2) => {};
 
+const lambdaInParamsFunctions = (fn = () => {}) => {};
```

Comment:
Could you also add a multiline example and an example with `(fn = function() {}})`?